### PR TITLE
Readds the cameras to the representative/consular office

### PR DESCRIPTION
**Pull request by Arrow768**
... due to recent events,

### CODE REVIEW

File: maps/aurora/aurora-4_mainlevel.dmm [hunk 1]
```diff
@@ -63832,17 +63836,6 @@
 /obj/structure/cable/green{
 	icon_state = "1-8"
 	},
-/obj/machinery/ringer_button{
-	id = "Representative";
-	name = "Representative ringer button";
-	pixel_x = -24;
-	pixel_y = 32
-	},
```
Comment (Arrow768) [inline maps/aurora/aurora-4_mainlevel.dmm:63840]: Seems like I accidentally removed that. Will be readded.

**Comment by Amunak** [comment]
You might want to remove the redundant camera in the briefing room.

**Comment by fernerr** [comment]
As discussed in devchat, I disagree with this pr.
Also, unless I'm missing something, 30 days has not passed since #10332 was merged for this to be a valid revert yet.

**Review by Amunak — APPROVED** [review]
I still think we should either keep the cams out of those offices (for gameplay reasons) or - as discussed - move to some more _robust_ camera system, but technically this looks alright.

**Comment by NonQueueingMatt** [comment]
Not really a fan of adding more cameras, so I'll have to disagree on this.

**Review by JohnWildkins — APPROVED** [review]
Technically fine but sounds like there's a lot of contention about it so we might want to look at feedback or have a, ahem, _robust_ discussion about this.

**Pull request merged**